model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -145,20 +145,25 @@ case class Workspace(
   lazy val workspaceIdAsUUID: UUID = UUID.fromString(workspaceId)
 }
 
+/** convenience constructor (for unit tests only!)
+  * defaults workspace version to v2 and google project id and google project number to random strings
+  * TODO: to be refactored/removed in https://broadworkbench.atlassian.net/browse/CA-1128
```

Comment:
based on discussion with Marc, we want to revisit this after PPW migration is complete